skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -92,10 +123,32 @@ def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
     
     See Also
     --------
-    skimage.measure.marching_cubes
+    skimage.measure.marching_cubes_classic
     skimage.measure.mesh_surface_area
     """ 
     
+    # This signature (output args) of this func changed after 0.12
+    try:
+        nout = expected_output_args()
+    except Exception:
+        nout = 0
+    if nout <= 2:
```

Comment:
Yes, if someone catches the result as a tuple, we have no way of knowing whether the change is going to break their code. Better safe than sorry, and warn in this case:
```python
mc_outputs = marching_cubes(...)
# ... somewhere later in the code
vertices, faces = mc_outputs
```

BTW: people could write compatible code (that does not raise the warning) by using `vertices, faces, *_ = marching_cubes(...)`.